environment_setup/setup_software.sh

```diff
@@ -29,29 +29,23 @@ echo "================================================================"
 echo "Setting up your shell config files"
 echo "================================================================"
 # Shell config files that various shells source when they run.
-# This is where we want to add aliases, source ROS environment
-# variables, etc.
+# This is where we want to add aliases, etc.
 SHELL_CONFIG_FILES=(
     "$HOME/.bashrc"\
     "$HOME/.zshrc"
 )
 
 # All lines listed here will be added to the shell config files
 # listed above, if they are not present already
-# These automatically perform functions outlined in the ROS setup tutorial
-# http://wiki.ros.org/melodic/Installation/Ubuntu
 declare -a new_shell_config_lines=(
-    # Source the ROS Environment Variables Automatically
-    "source /opt/ros/melodic/setup.sh"\
     # Source the setup script for our workspace. This normally needs to
     # be done manually each session before you can work on the workspace,
     # so we put it here for convenience.
     "source $GIT_ROOT/devel/setup.sh"\
     # Aliases to make development easier
     # You need to source the setup.sh script before launching CLion so it can
     # find catkin packages
-    "alias clion=\"source /opt/ros/melodic/setup.sh \
-        && source $GIT_ROOT/devel/setup.sh \
+    "alias clion=\"source $GIT_ROOT/devel/setup.sh \
```

Comment:
`devel` is gone, we shouldn't need this alias any more! :smile:

Reply:
done

Reply:
We can remove the `clion` alias entirely actually!

Reply:
Could leave it if ppl want an alias for clion disown exit

Reply:
Just my $0.02, it's probably worth removing. IMO enough people found it "weird" and a non-standard way of launching stuff that I think not having it would be the more popular option.

Reply:
Lol alright

Reply:
Yeah, I'm with Mat on this one. The less custom stuff (like aliases and whatnot) that we add to peoples computers the better.

Reply:
We don't _remove_ from the `.bashrc` though, so everyone who already had the alias will still have it.

Reply:
ok